### DISCUSSION TITLE
Proper way to inflate a class which expect an array

### DISCUSSION THREAD

**Discussion by SamueleD78** [vfsfitvnm/frida-il2cpp-bridge#191] [Q&A]
I have a class  ```System.Collections.Generic.Dictionary<System.String,System.String[]>```

In order to inflate it, passing the first argument it's really easy, i just recover the class and i pass to the inflate method as the first argument.
But what about the second, the array? Is it just the normal class or i have to do something more?
I'm in the dark.. thank you so much!

**Comment by vfsfitvnm** [comment]
An array class (e.g. `System.String[]`) is just a class like any other. There's a handy getter that returns the array class of a given class:
```ts
const SystemString = Il2Cpp.Image.corlib.class("System.String");
const SystemStringArray = SystemString.arrayClass; // System.String[]
```

**Reply by SamueleD78** [reply]
you are the best, you know that?
Thank you so much! ^^
